app/authorization_test.go

```diff
@@ -34,3 +37,167 @@ func TestCheckIfRolesGrantPermission(t *testing.T) {
 	}
 
 }
+
+// TestChannelRolesGrantPermission tests the system roles only, all of the channel roles are tested in
+// TestGetRolesByNamesAndTestChannelRolesGrantPermission.
+func TestChannelRolesGrantPermission(t *testing.T) {
```

Comment:
Can we incorporate everything that this is testing into a few lines of the roles test?